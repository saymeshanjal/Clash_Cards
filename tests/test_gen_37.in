100
14 128 3
16 195 5
8 88 5
12 157 5
2 195 1
4 129 4
12 144 2
2 28 5
10 137 5
4 32 1
8 166 5
10 189 2
16 160 5
10 190 5
12 94 2
12 47 1
8 4 3
14 85 1
14 65 5
10 74 2
8 101 5
8 58 4
16 25 5
6 142 2
8 19 1
16 177 5
12 20 5
8 34 2
12 8 2
2 140 1
10 16 5
12 19 3
10 42 5
8 29 4
2 32 4
8 104 3
12 173 3
10 32 2
2 13 3
12 5 3
12 89 1
6 183 1
4 75 5
4 30 4
2 134 4
6 185 5
12 78 3
10 123 2
12 3 2
16 137 2
14 46 4
10 136 2
14 113 4
8 106 4
10 10 5
14 140 5
2 129 3
16 45 3
14 171 2
6 122 4
10 184 2
16 29 3
4 123 3
14 14 1
4 179 2
6 105 4
10 146 4
10 73 4
2 176 2
16 114 5
16 94 1
12 109 5
8 142 1
6 116 2
8 117 3
16 127 4
2 131 5
6 160 5
4 136 1
10 167 1
4 86 2
14 43 4
4 22 1
16 40 5
8 3 2
8 75 2
10 195 3
16 45 3
8 118 1
2 90 3
16 131 1
10 130 4
16 163 1
4 58 1
16 29 2
6 8 2
12 52 2
10 3 4
6 191 2
4 39 3
