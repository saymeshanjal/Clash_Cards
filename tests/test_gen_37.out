1652 1780 0
4248 8622 0
8 62 0
219 705 0
1 1 0
0 5 1
286 287 351
0 2 0
45 207 0
3 3 0
8 62 0
76 77 99
4248 8622 0
45 207 0
286 287 351
462 462 0
1 69 0
1716 1716 0
981 2451 0
76 77 99
8 62 0
8 42 20
174 12696 0
5 6 9
35 35 0
4248 8622 0
15 909 0
20 21 29
49 277 598
1 1 0
2 250 0
98 826 0
45 207 0
8 42 20
0 2 0
27 43 0
430 494 0
71 72 109
0 2 0
28 896 0
462 462 0
10 10 0
0 6 0
0 5 1
0 2 0
1 19 0
430 494 0
76 77 99
84 462 378
4081 4082 4707
128 1884 1420
76 77 99
910 1520 1002
8 42 20
2 250 0
987 2445 0
0 2 0
3018 9852 0
1078 1079 1275
1 14 5
76 77 99
2123 10747 0
1 5 0
1554 1878 0
1 2 3
1 14 5
44 133 75
44 133 75
0 1 1
4248 8622 0
6435 6435 0
219 705 0
35 35 0
5 6 9
27 43 0
3809 5406 3655
0 2 0
1 19 0
3 3 0
126 126 0
1 2 3
94 2151 1187
3 3 0
533 12337 0
5 35 30
20 21 29
110 142 0
3018 9852 0
35 35 0
0 2 0
6435 6435 0
44 133 75
6435 6435 0
3 3 0
269 1241 11360
0 5 15
286 287 351
0 231 21
5 6 9
1 5 0
